nixos/modules/services/networking/sslh.nix

```diff
@@ -44,30 +41,30 @@ in
       '';
     };
 
-    listenAddresses = mkOption {
-      type = with types; coercedTo str singleton (listOf str);
+    listenAddresses = lib.mkOption {
+      type = with lib.types; coercedTo str lib.singleton (listOf str);
```

Comment:
`lib.singleton` was added by hand

Reply:
`singleton` was there before, so the only change is adding `lib.`